src/ECS/Registry.cpp

```diff
@@ -34,4 +34,8 @@ void Registry::SetDirty()
 {
 	Locator::rendereringSystem::value().SetDirty();
 }
+void Registry::Destroy(entt::entity entity)
```

Comment:
move it to the header like the rest of the functions unless there's a good reason